packages/instrumenter/src/transformers/directive-bookkeeper.ts

```diff
@@ -59,12 +64,28 @@ export class DirectiveBookkeeper {
       .filter(notEmpty)
       .forEach(([, directiveType, scope, mutators, optionalReason]) => {
         const mutatorNames = mutators.split(',').map((mutator) => mutator.trim().toLowerCase());
+
+        const directives = mutators
+          .split(',')
+          .map((mutator) => mutator.trim())
+          .filter((mutator) => mutator !== WILDCARD);
+        for (const directive of directives) {
+          if (!allMutators.map((x) => x.name.toLowerCase()).includes(directive.toLowerCase())) {
+            const mutant = new Mutant(directive, originFileName, node, {
+              mutatorName: directive,
+              ignoreReason: `Unused 'Stryker ${directiveType}' directive`,
+              replacement: node,
+            });
+            collector.collect(originFileName, node, mutant, { line: -1, position: 0 }); // Assumption that the directive is always -1 above it...
```

Comment:
Fair point, as I previously mentioned I was misusing them to get them in the report, so that they could get highlighted 🙈 . 

Would you rather inject a logger here, or a function and call it through all the layers? Option 1 probably will log `WARN DirectiveBookkeeper` rather than `WARN Instrumenter`?

Reply:
We don't _really_ use the logger categories ATM. I'd be happy with it just being `WARN Instrumenter`. That seems the easiest and straight forward way for now.